css/properties/offset-path.json

```
@@ -44,7 +44,7 @@
               }
             ],
             "safari": {
-              "version_added": false
+              "version_added": "15.5"
```

Looks like this should actually be 15.4 based upon https://bugs.webkit.org/show_bug.cgi?id=231801 and https://github.com/WebKit/WebKit/commit/d3d6c2754702263730e05ec7627746d1f0b766f7 (see [Version.xcconfig](https://github.com/WebKit/WebKit/blob/d3d6c2754702263730e05ec7627746d1f0b766f7/Source/WebKit/Configurations/Version.xcconfig) which indicates this was added in WebKit 613.1.8, where Safari 15.4 uses WebKit 613.1.17).

```suggestion
              "version_added": "15.4"
```